shared/constants/settings.js

```diff
@@ -23,7 +43,29 @@ export type DeleteAccountForever = NoErrorTypedAction<'settings:deleteAccountFor
 export type PlanLevel = 'Basic' | 'Gold' | 'Friend'
 const plans: Array<PlanLevel> = ['Basic', 'Gold', 'Friend']
 
-export type Actions = NotificationsRefresh | NotificationsRefreshed | NotificationsSave | NotificationsSaved | NotificationsToggle | SetAllowDeleteAccount
+export type PaymentInfo = {
```

Comment:
I moved this type out into constants/plan-billing. Do you need it here? or was this a bad merge?